flyway/migrations/V5__Create_SittingJobTable.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE SittingJob (
+    PK_jobID uniqueidentifier DEFAULT NEWSEQUENTIALID() PRIMARY KEY,
+    FK_sitterID uniqueidentifier NOT NULL,
+    FK_clientID uniqueidentifier NOT NULL,
+    startDate DATE,
+    startTime TIME,
+    endDate DATE,
+    endTime TIME,
+    jobRate INT,
```

Comment:
Does not match ERD
Name of column should be TotalCost and of the type DECIMAL.
![image](https://github.com/House-and-Pet-Sitting-Database-Levelup/HouseAndPetSittingDB/assets/125252284/40a5beef-ed62-49c5-84b2-7998c53edf60)